src/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #000;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px;
+}
+
+.card__title {
+  margin-top: 45px;
+  margin-bottom: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin-top: 6px;
+}
+
+.card__rating {
+  margin-top: 14px;
+  display: inherit;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: inherit;
+  justify-content: space-between;
+}
+
+.card__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  display: inline-block;
+}
+
+.card__star:nth-child(-n+4) {
```

Comment:
I'm still waiting for all 5 classes(which represents all situations, when we have 1 active stars 2, 3 and so on )